README.md

```diff
@@ -141,15 +144,15 @@ spinnaker_instance=`aws cloudformation describe-stacks --region us-west-2 --stac
 
 Sync your changes to the development instance:
 ```
-rsync --progress -a ~/code/spinnaker/ ubuntu@$spinnaker_instance:/home/ubuntu/dev/spinnaker
+rsync --progress -a ~/code/spinnaker/ ubuntu@$spinnaker_instance:/home/ubuntu/dev/spinnaker -i /path/to/my-key-pair.pem
```

Comment:
Is that a real option? Is this a Linux vs Mac difference? When I do `man rsync`, I see
```
-i, --itemize-changes       output a change-summary for all updates
```

Reply:
Whoops looks like I didn't commit the whole command! Fixed!